.github/workflows/release-body.yml

```diff
@@ -42,21 +43,78 @@ jobs:
 
           # Safeguard: you need to both set "latest" in GH and not have suffixes to overwrite latest
           is_latest = '${{ steps.latest_release.outputs.release }}' == tag_name and not is_prerelease
+          
+          is_driver = "+driver" in tag_name
```

Comment:
This is enough since we already have the `is_latest` flag above, and we are already sure that the tag follows the conventional regex.